### DISCUSSION TITLE
Workshop closing poll

### DISCUSSION THREAD

**Discussion by rebecca-vanhoeck** [SAEL-SWFSC/PAM-SI-Mtg-March2024#88] [Q&A]
How did this workshop go? What did you like? Is there anything we should have spend more or less time on?

**Comment by jlkeating** [comment]
Very much liked the opportunity to work in person with everyone. The working groups were well structured and having the format for project/action items really helped keep us on task.

**Comment by shannonrankin** [comment]
Very happy with the collaboration, ideas, and (mostly) realistic action items

**Comment by TaikiSan21** [comment]
I thought it was really well organized, and I think the round table discussions in particular were really productive. I'm excited to get to work helping everyone!

**Comment by ybarkley** [comment]
I think the meeting went really well and was a great experience to learn what other PAM programs are working on and the similarities between all of them. The brainstorming times were productive and insightful. It's also helpful to understand what common challenges we are all facing and how we can move forward as a larger group.

**Comment by cberchok** [comment]
It was so nice seeing everyone again, and meeting the new folks!   I think we've come up with a good combination of attainable and aspirational ideas!

**Comment by GenevieveDavis** [comment]
Fantastic. Really great ideas, really great plans for collaboration and working toward common goals. Wonderful to get time with everyone. Thoguht the structure and flow was great!

**Comment by namariD** [comment]
I really liked how this workshop was organized and appreciate all the thought the facilitators put into it. Definitely would have liked to do more of the speed dating, I don't think I got to talk to everyone and a lot of great conversations came out of it.

**Comment by ludovictenorio** [comment]
Great workshop! Looking forward to keep working with all of you towards common goals. In-person time was invaluable to get to know everyone better and facilitate working together moving forward. Thank you to all the organizers!
